tests/by_image/base-notebook/test_ips.py

```diff
@@ -38,13 +38,9 @@ def test_ipv46(container: TrackedContainer, ipv6_network: str) -> None:
     host_data_dir = THIS_DIR / "data"
     cont_data_dir = "/home/jovyan/data"
     LOGGER.info("Testing that server is listening on IPv4 and IPv6 ...")
-    running_container = container.run_detached(
+    container.run_detached(
         network=ipv6_network,
         volumes={str(host_data_dir): {"bind": cont_data_dir, "mode": "ro,z"}},
         tty=True,
     )
-
-    command = ["python", f"{cont_data_dir}/check_listening.py"]
```

Comment:
For simplicity made `cmd` always a string.
python docker allows both list or str, having str loogs good for us.

Additional improvement - if something goes wrong, copy-paste command will be easier